pkg/interactive/roles/roles.go

```diff
@@ -44,13 +44,13 @@ func GetInstallerRoleArn(r *rosa.Runtime, cmd *cobra.Command,
 			}
 		}
 		r.Reporter.Warnf("More than one %s role found", role.Name)
-		if !Enabled() && confirm.Yes() {
+		if !interactive.Enabled() && confirm.Yes() {
```

Comment:
Needless changes by removing `.` from the import. The usage of `.` here is correct, and results in a nicer-to-work-with representation of the parent package